pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -609,6 +619,84 @@ def visit_if(self, node):
         self._check_superfluous_else_break(node)
         self._check_superfluous_else_continue(node)
         self._check_consider_get(node)
+        self._check_consider_using_min_max_builtin(node)
+
+    def _check_consider_using_min_max_builtin(self, node):
```

Comment:
```suggestion
    def _check_consider_using_min_max_builtin(self, node: astroid.Node) -> None:
```
Just a suggestion, I'm not sure about the node type

Reply:
I think astroid.node_classes.NodeNG might be the one you mean, but the other functions nearby don't have hinting in their signature. Do you think it's better for us to keep with the uniformity or add the hints?

Reply:
We do not have typing because we were compatible with Python 3.5 until the 2.7 version. But now we can add it progressively :)